include/scippp/model.hpp

```diff
@@ -94,8 +95,8 @@ class Model {
         const std::string& name,
         SCIP_Real coeff = 0.0,
         VarType varType = VarType::CONTINUOUS,
-        SCIP_Real lb = 0.0,
-        SCIP_Real ub = 1.0);
+        std::optional<double> lb = 0.0,
+        std::optional<double> ub = 1.0);
```

Comment:
Please also add something like `\c std::nullopt is interpreted as infinity.` to the docu